package.json

```diff
@@ -77,7 +78,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run security-checks",
+      "pre-commit": "npm run prepush && npm run security-checks",
```

Comment:
doesn't prepush already call security-checks. I think this will make you call security-checks twice.